index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+
+<html lang="eng">
+<head>
+    <title>Portfolio Website</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://use.typekit.net/bgu6sml.css">
+    <link href="normalize.css" type="text/css" rel="stylesheet">
+    <link href="styles.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+
+<div class="wrapper-content">
+
+    <header>
+        <nav>
+            <ul>
+                <li><a class="about-link" href="#about">about</a></li>
+                <li><a class="projects-link" href="#projects">projects</a></li>
+                <li><a class="contact-link" href="#contact">contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        
+        <section class="homepage">
+            <h1>Hi, I'm Rosa,
+               <span class="subtitle">a <span class="highlight-subtitle">(soon-to-be)</span> full stack developer</span> 
+                <span class="subtitle">based in Bristol.</span>
+            </h1>
+        </section>
+
+        <section id="about">
+            <h2 class="about-title">about</h2>
+            <div class="about-content-container">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <img class="photo-rosa" src="rosa-placeholder.jpeg" alt="a photo of Rosa">
+            </div>
+        </section>
+
+        <section id="projects">
+            <h2 class="projects-title">projects</h2>
+            <div class="projects-container">
+                <a href="#" class="projects project1">
+                    <div class="project-images project-image1"></div>
+                </a>
+                <a href="#" class="projects project2">
+                    <div class="project-images project-image2"></div>
+                </a>
+                <a href="#" class="projects project3">
+                    <div class="project-images project-image3"></div>
+                </a>
+                <a href="#" class="projects project4">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project5">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project6">
```

Comment:
as far as i can tell the classes project1 to project6 and project-image1 to project-image6 are not doing anything? if so, you can remove them. also if there are any other classes that are not used for anything they should be removed as well - we create css classes only when they are needed

if they are doing something - bare in mind the class on the 5th project image is project-image4 - probably a mistake